app/util/session_util.py

```diff
@@ -28,3 +28,13 @@ def decorated_function(*args, **kwargs):
             return redirect(url_for('login'))
         return f(*args, **kwargs)
     return decorated_function
+
+
+def abort_if_not_current_user(f):
```

Comment:
Every page this wrapper would be used, the user would have a valid session. Rather than writing @login_required and @abort_if_not_current_user every time, it might make more sense to call login_required from this function.

Reply:
I think we should have consistent naming for decorators, and especially for reuse, having the name describe at a high level what it's accomplishing (authorization) rather than how it's accomplishing it (aborts if ...). To that end, it should be called 'authorization_required'.